front_end/html/unavailable_assignment.html

```diff
@@ -7,6 +7,8 @@ <h2>{{ assignment_basics["title"] }}</h2>
             <p>This assignment will open on {{ start_date }}.</p>
         {% elif error == "due" %}
             <p>This assignment closed on {{ due_date }}.</p>
+        {% elif error == "restricted_ip" %}
+            <p>This assignment can only be completed from computers that have an allowed IP address. The IP address for this computer does not match any of the allowed addresses; please try logging on from a different machine.</p>
```

Comment:
Instead of "please try logging on from a different machine," please put "please consult the instructor for additional instructions."